packages/tracing/src/browser/metrics.ts

```diff
@@ -17,7 +17,7 @@ const global = getGlobalObject<Window>();
 
 /** Class tracking metrics  */
 export class MetricsInstrumentation {
```

Comment:
I'm not the biggest fan of this class - one day I'll tear it down lol

Reply:
Yeah agree. I feel like this class could be a function.